monai/data/dataset.py

```diff
@@ -264,7 +266,16 @@ def __init__(
                 When this is enabled, the traced transform instance IDs will be removed from the cached MetaTensors.
                 This is useful for skipping the transform instance checks when inverting applied operations
                 using the cached content and with re-created transform instances.
+            track_meta: whether to track the meta information, if `True`, will convert to `MetaTensor`.
+                default to `False`. Cannot be used with `weights_only=True`.
+            weights_only: keyword argument passed to `torch.load` when reading cached files.
+                default to `True`. When set to `True`, `torch.load` restricts loading to tensors and
+                other safe objects. Setting this to `False` is required for loading `MetaTensor`
+                objects saved with `track_meta=True`.
```

Comment:
```suggestion
                other safe objects. Setting this to `False` is required for loading `MetaTensor`
                objects saved with `track_meta=True`, however this creates the possibility of remote
                code execution through `torch.load` so be aware of the security implications of doing so.
```

I'd like to explicitly state the security concern here that motivated the previous changes to the datasets.